dockers/docker-sonic-restapi/restapi.sh

```diff
@@ -3,7 +3,10 @@
 RESTAPI_ARGS=""
 while true
 do
-    client_auth=`sonic-cfggen -d -v "RESTAPI['config']['client_auth']"`
+    has_client_auth=$(sonic-cfggen -d -v "1 if RESTAPI and RESTAPI['config']")
+    if [ "$has_client_auth" == "1" ]; then
+        client_auth=$(sonic-cfggen -d -v "RESTAPI['config']['client_auth']")
+    fi
     if [[ $client_auth == 'true' ]]; then
```

Comment:
Is it okay, if client_auth is not set? Which is going to happen in the else case of  [ "$has_client_auth" == "1" ]

Reply:
Yes, then it would evaluate to False